target/postfix/main.cf

```diff
@@ -101,5 +101,7 @@ policyd-spf_time_limit = 3600
 # Header checks for content inspection on receiving
 header_checks = pcre:/etc/postfix/maps/header_checks.pcre
 
-# Remove unwanted headers that reveail our privacy
+# Remove unwanted headers that reveal our privacy
 smtp_header_checks = pcre:/etc/postfix/maps/sender_header_filter.pcre
+
+compatibility_level = 2
```

Comment:
```suggestion
# Leaving Postfix compatibility_level at the default 0, when the config is changed (e.: via postconf), it will log the following each time:
#    Postfix is running with backwards-compatible default settings
#    See http://www.postfix.org/COMPATIBILITY_README.html for details
#    To disable backwards compatibility use "postconf compatibility_level=2" and "postfix reload"
compatibility_level = 2
```